articles/aks/rdp.md

```diff
@@ -23,128 +23,63 @@ This article assumes that you have an existing AKS cluster with a Windows Server
+aks-nodepool1-12345678-vmss000001   Ready    agent   13m     v1.19.9   10.240.0.35   <none>        Ubuntu 18.04.5 LTS               5.4.0-1046-azure   containerd://1.4.4+azure
+aksnpwin000000                      Ready    agent   87s     v1.19.9   10.240.0.67   <none>        Windows Server 2019 Datacenter   10.0.17763.1935    docker://19.3.1
 ```
 
-## Get the node address
-
-To manage a Kubernetes cluster, you use [kubectl][kubectl], the Kubernetes command-line client. If you use Azure Cloud Shell, `kubectl` is already installed. To install `kubectl` locally, use the [az aks install-cli][az-aks-install-cli] command:
-    
-```azurecli-interactive
-az aks install-cli
-```
+In the above example, *10.240.0.67* is the internal IP address of the Windows Server node.
 
-To configure `kubectl` to connect to your Kubernetes cluster, use the [az aks get-credentials][az-aks-get-credentials] command. This command downloads credentials and configures the Kubernetes CLI to use them.
+Use ssh to forward the RDP port of the Windows machine to your local network (you may need to change the first local port 3389 if it is being used on your computer):
 
-```azurecli-interactive
-az aks get-credentials --resource-group myResourceGroup --name myAKSCluster
 ```
-
-List the internal IP address of the Windows Server nodes using the [kubectl get][kubectl-get] command:
-
-```console
-kubectl get nodes -o wide
+ssh -p 2022 -L 3389:10.240.0.67:3389 -A azureuser@127.0.0.1
 ```
 
-The follow example output shows the internal IP addresses of all the nodes in the cluster, including the Windows Server nodes.
+## Connect to Windows node
 
-```console
-$ kubectl get nodes -o wide
-NAME                                STATUS   ROLES   AGE   VERSION   INTERNAL-IP   EXTERNAL-IP   OS-IMAGE                    KERNEL-VERSION      CONTAINER-RUNTIME
-aks-nodepool1-42485177-vmss000000   Ready    agent   18h   v1.12.7   10.240.0.4    <none>        Ubuntu 16.04.6 LTS          4.15.0-1040-azure   docker://3.0.4
-aksnpwin000000                      Ready    agent   13h   v1.12.7   10.240.0.67   <none>        Windows Server Datacenter   10.0.17763.437
-```
-
-Record the internal IP address of the Windows Server node you wish to troubleshoot. You will use this address in a later step.
-
-## Connect to the virtual machine and node
-
-Connect to the public IP address of the virtual machine you created earlier using an RDP client such as [Microsoft Remote Desktop][rdp-mac].
-
-![Image of connecting to the virtual machine using an RDP client](media/rdp/vm-rdp.png)
-
-After you've connected to your virtual machine, connect to the *internal IP address* of the Windows Server node you want to troubleshoot using an RDP client from within your virtual machine.
+Now that you have the RDP port forwarded, connect to the Windows Server node you want to troubleshoot using an RDP client from with your localaddress:
 
+[should update this image to be to 127.0.0.1 from a mac connection to be consistent]
```

Comment:
The other images showed a Mac ssh RDP program. This should probably match that. Would someone be able to upload a image like that here or is it ok to show a Linux RDP program?